finality-aleph/src/abft/mod.rs

```diff
@@ -72,6 +73,18 @@ impl<S: 'static> IntoIterator for SignatureSet<S> {
     }
 }
 
+impl From<SignatureSet<crate::crypto::Signature>> for ScoreSignatureSet {
+    fn from(signature_set: SignatureSet<crate::crypto::Signature>) -> ScoreSignatureSet {
+        let score_sigantures: Vec<(primitives::NodeIndex, primitives::AuthoritySignature)> =
+            signature_set
+                .0
+                .into_iter()
+                .map(|(idx, s)| (idx.0 as u64, s.into()))
```

Comment:
You could just construct `IndexedSignature` here directly, and then you don't need a lot of the boilerplate (one `Into`, the typing above probably if you just constructed `ScoreSignatureSet` below explicitly).

Reply:
Good idea!

Reply:
done